.gitignore

```diff
@@ -24,6 +24,9 @@ last-run-results.html
 *.report.pretty
 *.artifacts.log
 
+!lighthouse-cli/test/sample_artifacts/*.trace.json
```

Comment:
should these live somewhere with `sample_v2.json`?

Reply:
sure!

how about  `lighthouse-core/test/results/artifacts/`?